dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1743,6 +1743,664 @@
+          "description": "The numeric value that the extracted text denotes."
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "AgeResolution": {
+      "description": "Represents the Age entity resolution model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        },
+        {
+          "$ref": "#/definitions/QuantityResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "AgeResolution",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "enum": [
+            "Unspecified",
+            "Year",
+            "Month",
+            "Week",
+            "Day"
+          ],
+          "x-ms-enum": {
+            "name": "AgeUnit",
+            "modelAsString": true
+          },
+          "description": "The Age Unit of measurement"
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "VolumeResolution": {
+      "description": "Represents the volume entity resolution model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        },
+        {
+          "$ref": "#/definitions/QuantityResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "VolumeResolution",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "enum": [
+            "Unspecified",
+            "CubicMeter",
+            "CubicCentimeter",
+            "CubicMillimeter",
+            "Hectoliter",
+            "Decaliter",
+            "Liter",
+            "Centiliter",
+            "Milliliter",
+            "CubicYard",
+            "CubicInch",
+            "CubicFoot",
+            "CubicMile",
+            "FluidOunce",
+            "Teaspoon",
+            "Tablespoon",
+            "Pint",
+            "Quart",
+            "Cup",
+            "Gill",
+            "Pinch",
+            "FluidDram",
+            "Barrel",
+            "Minim",
+            "Cord",
+            "Peck",
+            "Bushel",
+            "Hogshead"
+          ],
+          "x-ms-enum": {
+            "name": "VolumeUnit",
+            "modelAsString": true
+          },
+          "description": "The Volume Unit of measurement"
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "SpeedResolution": {
+      "description": "Represents the speed entity resolution model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        },
+        {
+          "$ref": "#/definitions/QuantityResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "SpeedResolution",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "enum": [
+            "Unspecified",
+            "MeterPerSecond",
+            "KilometerPerHour",
+            "KilometerPerMinute",
+            "KilometerPerSecond",
+            "MilesPerHour",
```

Comment:
Should this be MilePerHour? @joseharriaga

Reply:
You're correct: It should be `MilePerHour`.

Reply:
fixed.